### PR TITLE
Fix typo in the template name

### PR DESCRIPTION
**Pull request by AR-May**
Issue #3725

### CODE REVIEW

File: packaging/Microsoft.Dotnet.Wpf.ProjectTemplates/content/WpfClassLibrary-CSharp/.template.config/template.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "$schema": "http://json.schemastore.org/template",
   "author": "Microsoft",
   "classifications": ["Common", "WPF"],
-  "name": "WPF Class library",
+  "name": "WPF Class Library",
   "generatorVersions": "[1.0.0.0-*)",
   "description": "A project for creating a class library that targets a .NET Core WPF Application",
```
Comment (lindexi) [inline packaging/Microsoft.Dotnet.Wpf.ProjectTemplates/content/WpfClassLibrary-CSharp/.template.config/template.json:7]: Should we replace `a .NET WPF` to `a .NET Core WPF` ?

We will update to dotnet 5

Comment (AR-May) [reply packaging/Microsoft.Dotnet.Wpf.ProjectTemplates/content/WpfClassLibrary-CSharp/.template.config/template.json:7]: Sounds reasonable, but I am not sure. @ryalanms, should we?

**Comment by predavid** [comment]
@ryalanms - let's take a look at this.

**Review by fabiant3 — APPROVED** [review]

File: packaging/Microsoft.Dotnet.Wpf.ProjectTemplates/content/WpfClassLibrary-CSharp/.template.config/template.json (hunk 1 above)
Comment (ryalanms) [reply packaging/Microsoft.Dotnet.Wpf.ProjectTemplates/content/WpfClassLibrary-CSharp/.template.config/template.json:7]: Hi @AR-May.  I agree with @lindexi that .NET Core should be updated to .NET.  I will merge your typo fix for now, and either of you can create another PR for any additional updates.  Thanks.

**Pull request merged**